src/vivarium_census_prl_synth_pop/components/businesses.py

```diff
@@ -168,11 +171,14 @@ def on_time_step(self, event: Event):
             address_id_col_name="employer_address_id",
         )
 
-        # change jobs if of working age already
+        # change jobs if of working age already and either selected by rate or moved
+        # on this timestep
         working_age_idx = pop.loc[pop["age"] >= data_values.WORKING_AGE].index
         changing_jobs_idx = filter_by_rate(
             working_age_idx, self.randomness, self.job_change_rate, "changing jobs"
         )
+        moved_idx = pop.index[pop["address_id"] != pop["previous_timestep_address_id"]]
+        changing_jobs_idx = changing_jobs_idx.union(moved_idx.intersection(working_age_idx))
```

Comment:
So every unemployed person who moves will get a job?

Reply:
Yes -- though I have made a TODO to spell this out more as a limitation in the docs. It's not accurate (obviously) but shouldn't be super important for PRL purposes.

Reply:
I think this will give jobs to (unemployed) people under 18 who move. Even if it doesn't you should explicitly test that you aren't giving them jobs.

Reply:
This test checks for that: https://github.com/ihmeuw/vivarium_census_prl_synth_pop/pull/127/files#diff-5188a0bafecb0feeccbd82bdd91b9bf65eb54464a68982faaa9f5e542d1c6c11R27-R31

The code here doesn't give people under 18 jobs because of the `.intersection(working_age_idx)` but I can separate that into a named variable instead of doing it inline for clarity.